initializers/custom_links.yml

```diff
@@ -0,0 +1,21 @@
+## Possible Choices:
+##   new_window:
+##   - True
+##   - False
+##   content_type:
+##   - device
+##   - site
+##   - any-other-content-type
+##
+## Examples:
+
+#  - name: link_to_repo
+#    text: 'Link to docker repository'
+#    url: 'https://github.com/netbox-community/netbox-docker'
+#    new_window: False
+#    content_type: device
+#  - name: link_to_localhost
+#  text: 'Link to the users localhost'
+#  url: 'http://localhost'
+#  new_window: True
+#  content_type: device
```

Comment:
This part is still wrong:

```suggestion
#  - name: link_to_localhost
#    text: 'Link to the users localhost'
#    url: 'http://localhost'
#    new_window: True
#    content_type: device
```

You can run the test locally to see if your changes work and integrate with all the other initializers:

```bash
./build-latest.sh && ./test.sh latest
```